tests/pip.bats

```diff
@@ -134,3 +134,21 @@ setup() {
     [[ $status = 0 ]]
     [[ ${lines[${#lines[@]}-1]} = $'pty_copy\r' ]]
 }
+
+@test "py3: recursive requirements files" {
+    echo 'urp' > /work/tests/pip/temp1.txt
```

Comment:
Don't write files in the test dir, unless there is no way out.  You need just two one line files committed to git:
```
include-short.txt: -r requirements.txt
include-nested.txt: --requirement include-short.txt 
```
This assumes files are in `tests/pip` and use `tests/pip/requirements.txt` that contains 'urp'. And you run urp as in all other tests to ensure it's installed.